client/js/libs/handlebars/tz.js

```diff
@@ -1,8 +1,10 @@
+var timestamp = window.timestamp;
+
 Handlebars.registerHelper(
 	"tz", function(time) {
 		if (time) {
-			var utc = moment.utc(time, "HH:mm:ss").toDate();
-			return moment(utc).format("HH:mm");
+			var utc = moment.utc(time, timestamp).toDate();
```

Comment:
Changing "HH:mm:ss" to variable shouldn't be done here probably? As that's what the server sends.